Lib/test/test_typing.py

```diff
@@ -7319,6 +7319,17 @@ def test_re_submodule(self):
             self.assertEqual(__name__, 'typing.re')
             self.assertEqual(len(w), 1)
 
+    def test_re_submodule_access_basics(self):
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
Is the `w` variable here used at all?

```suggestion
        with warnings.catch_warnings():
```